assets/xml/overlays/ovl_Oceff_Wipe7.xml

```diff
@@ -0,0 +1,10 @@
+<Root>
+    <File Name="ovl_Oceff_Wipe7" BaseAddress="0x80BCDCB0" RangeStart="0x330" RangeEnd="0xDB8">
+        <Texture Name="gOceff7Tex" OutName="oceff7" Format="i8" Width="32" Height="32" Offset="0x330"/>
+        <TextureAnimation Name="ovl_Oceff_Wipe7_Matanimheader_000B38" Offset="0xB38" />
+        <Array Name="gOceff7Vtx" Count="22" Offset="0xB40">
+            <Vtx/>
+        </Array>
```

Comment:
Looks like you should be fine just extracting these implicitly.

Reply:
it is used in the draw function